platforms.sh

```diff
@@ -827,3 +827,11 @@ HPC_Reservation_Hint()
    esac
 }
 
+localPlatformParameters=$SCRIPTDIR/local/platforms.sh
```

Comment:
I would wait to do the platforms.sh, since it has some things in there that are pretty specific and idk yet how we'd implement those generally (`HPCENV`, the _Hints_, etc)

Reply:
Ok I will comment that out.